app/pages/project/stats/stats.cjsx

```diff
@@ -164,9 +164,9 @@ ProjectStatsPage = React.createClass
   render: ->
     progress = @workflowInfo()
     #Dates for gap in classification stats
-    classificationGap = ['2016-01-13T00:00:00.000Z', '2016-02-07T00:00:00.000Z']
+    classificationGap = ['2015-06-30T00:00:00.000Z', '2016-06-09T00:00:00.000Z']
```

Comment:
- should the first date's year be `2016`? 

- Also, previously it went earlier date then later date. If the first date's year above should be `2015` we're good, but if it should be `2016` should the dates be flip-flopped?